docs/topics/transforms.rst

```diff
@@ -69,17 +69,21 @@ a value of 0 is assumed.
             transformer.xy(0, 0)
     (-123.47959047080701, 49.52794990575094)
 
-A first order correction would be to use a mean elevation value for the image
+A constant height offset can be specified using the ``rpc_height`` keyword argument. This is useful 
+for datasets with little elevation change. In this case, ``rpc_height`` is assumed to be an average
+height above sea level for ground in the target scene, while ``zs`` is the height above ground.
```

Comment:
**Nitipick** The sentence could be made a bit more clear. E.g.
```
In this case, ``rpc_height`` is assumed to be an average
height above sea level for the target scene, while ``zs`` is the height difference between actual ground and the average height.
```
The `zs` could be positive or negative.